### DISCUSSION TITLE
启动容器后访问页面报 401

### DISCUSSION THREAD

**Discussion by zooah212** [whyour/qinglong#1962] [Q&A]
华为云服务器，开放了5600端口
安装了docker，启动了容器，访问http://{ip}:5600，不显示页面，只显示如下信息
{"code":401,"message":"No authorization token was found"}

大佬们这是什么问题，试了几次都这样。。

**Comment by zooah212** [comment]
找了一个之前备份的青龙docker镜像恢复了一下，能正常访问，是新版本的问题吗？

**Reply by Mr9357** [reply]
我刚刚用宿主机更新命令，也这样了

**Comment by Mr9357** [comment]
同样问题。我用没登陆过的手机来登陆，能正常登陆，再刷新，就又出现401错误提示。我设置的机器人也正常发来了成功登陆通知。

**Comment by alanwozc** [comment]
将端口改成5700后成功启动
